blocked-edges/4.14.0-ARODNSWrongBootSequence.yaml

```diff
@@ -0,0 +1,15 @@
+to: 4.14.0
+from: .*
+url: https://issues.redhat.com/browse/OCPBUGS-35300
+name: ARODNSWrongBootSequence
+message: |-
+  Disconnected clusters or clusters with a UDR 0.0.0.0/0 route definition that are blocking the ARO ACR and quay, are not be able to add or replace nodes after an upgrade
+matchingRules:
+- type: PromQL
+  promql:
+    promql: |
+      (
```

Comment:
nit: no need for these wrapping parents, see [this example][1].

[1]: https://github.com/openshift/cincinnati-graph-data/blame/ed47a1518646381ff54fdac170d51ecdeed17274/README.md#L190-L192

Reply:
*sigh* I used a script given to me and left it as found. I'll fix.

Reply:
2c351d82177ebfe1 addressed this; marking this thread resolved.